dividers/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=dividers
+POM_NAME=Vitamin Dividers
+POM_DESCRIPTION=Divider components with Vitamin design
+VERSION_NAME=0.1.0-alpha04
```

Comment:
`VERSION_NAME` is now located in `gradle.properties` (root).

Reply:
Alright I removed the version in dividers/gradle.properties